checks/runtimeCheck_test.go

```diff
@@ -0,0 +1,25 @@
+package checks
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRuntimeCheck(t *testing.T) {
+	dirname, _ := os.Getwd()
```

Comment:
Here, too, I think we should check for this error with an `assert.Nil(err)` and assert that this `dirName` has a value other than "" to be safe.